api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -352,14 +352,14 @@ class ExperimentDataPipe(pipes.IterDataPipe[Dataset[ObsAndXDatum]]):  # type: ig
                  [2416,    0,    4],
                  [2417,    0,    3]], dtype=torch.int64))
 
-    The ``return_sparse_X`` parameter controls whether the ``X`` data is returned as a dense or sparse Tensor.  If the
-    model supports use of sparse Tensors, this will reduce memory usage.
+    The ``return_sparse_X`` parameter controls whether the ``X`` data is returned as a dense or sparse
+    :class:`torch.Tensor`.  If the model supports use of sparse :class:`torch.Tensor`s, this will reduce memory usage.
```

Comment:
Extra space after period in:  `.  If the`

Reply:
Addressed